_episodes/55.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: episode
+title: "55: Commander Bun Bun peaks at Peaka"
+date: 2024-01-18
+tags: trino peaka
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "Episode 55: Commander Bun Bun peaks at Peaka"
+  time: 0
+- title: "Trino releases"
+  desc: "437-438"
+  time:
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Mustafa Sakalsiz](https://linkedin.com/in/sakalsiz), CEO at
+  [Peaka](https://www.peaka.com/)
+* [Ali Tekin](https://www.linkedin.com/in/alitekin/), Principal Software
+  Architect at [Peaka](https://www.peaka.com/)
+
+## Releases 437-438
+
+[Trino 437](https://trino.io/docs/current/release/release-437.html)
+
+* Support for configuring compression codecs
+* Support for `char` values in the `to_utf8()` and `lpad()` functions
+* Improved performance for Delta Lake queries without table statistics
+* Improved performance for Iceberg queries with filters on `ROW` columns
+
+[Trino 438](https://trino.io/docs/current/release/release-438.html)
+
+* Support for access control with Open Policy Agent
+* Support for `ALTER COLUMN ... DROP NOT NULL` in Iceberg and PostgreSQL
+* Support for configuring page sizes in Delta Lake, Hive, and Iceberg
+* Better type support for the `reduce_agg()` function
+
+## Concept of the episode: Peaka
+
+Another Trino Community Broadcast episode means another cool piece of technology
+that works with Trino for us to show off to the community. This time it's Peaka,
+a no-code approach to date warehousing that makes it easier than ever to set up
+your data stack without needing a ton of complex engineering.
```

Comment:
you are not really setting up a stack .. Peaka kinda is the stack .. right? Apart from data sources I guess

Reply:
Yes, Peaka itself is a data stack. The text works for us, very nice.